setup.py

```diff
@@ -39,6 +39,39 @@ def get_version():
         shutil.rmtree(dirpath)
 
 
+# Remove symlinks as Python wheels do not support them at the moment.
```

Comment:
I think if you remove the `SOVERSION` it shuld fix the problem. And anyway when building the wheel, it makes little sense to have the version embedded in the dynamic library file name.

Reply:
I made a flag to disable the symlink, this is much simpler indeed, thanks.

Reply:
Indeed, it's now much cleaner!